ansible/README_DISTRIBUTED.md

```diff
@@ -15,10 +15,8 @@ Login to your bootstrapper VM. Your local machine can act as the bootstrapping m
 This installs modules and packages to manage cloud instances via Ansible.
 
 ```
-sudo apt-get -y install python-setuptools python-dev libssl-dev
-sudo pip install shade pytz positional appdirs monotonic rfc3986
-sudo apt-get install python-novaclient
-sudo pip install six --upgrade
+sudo apt-get -y install python-setuptools python-dev libssl-dev build-essential libssl-dev libffi-dev python-dev python-novaclient
+sudo pip install shade pytz positional appdirs monotonic rfc3986 pyparsing stevedore debtcollector netaddr oslo.config futures warlock six
```

Comment:
One idea to explore to make it easier is to try to move these commands into the `setup.yaml` or install them from a `dependencies.txt` file as: `pip install -r dependencies.txt`, also controlling the versions.